addons/mrp/models/mrp_production.py

```diff
@@ -222,6 +222,7 @@ def _get_default_date_planned_finished(self):
     picking_ids = fields.Many2many('stock.picking', compute='_compute_picking_ids', string='Picking associated to this manufacturing order')
     delivery_count = fields.Integer(string='Delivery Orders', compute='_compute_picking_ids')
     confirm_cancel = fields.Boolean(compute='_compute_confirm_cancel')
+    consumption = fields.Selection(related='bom_id.consumption', store=True)
```

Comment:
it sould be better that this field is completed when you create the `mrp.production` (by create methode or by an onchange of bom_id). If not when you should change the consuption in the bom it will change on all allready existing `mrp.production`.